equality.js

```diff
@@ -0,0 +1,2 @@
+const strictEqual = (a,b) => a == b && typeof a == typeof b
```

Comment:
now that we are out of html and in node projects start referring to your js as index.js unless you learn something different.